db/migrations/development/03_create_ingredients_tables.js

```diff
@@ -0,0 +1,14 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('ingredients', t => {
+        t.increments('id')
+        t.string('igredient_name')
+        t.integer('quantity')
+        t.string('unit')
+        t.string('location')
```

Comment:
what is this location column for?

Reply:
This is for when producing groceries list with prices. It's a nice to have feature, that w e will likely not implement. But a user at some point could note: onions cost me $a.bc at stop n shop, but $d.ef at market basket. The grocery list could then split the items into two different lists. We won't use this column I believe, just included it in case we ever did get there